.gitignore

```diff
@@ -0,0 +1,2 @@
+# Ignoring saved wallpapers
+APOD_images/**
```

Comment:
Also, I don't think you need the ** part, just the name of the directory is enough to ignore everything under it. I think even without the / will work... try it...